picard/ui/options/renaming.py

```diff
@@ -22,11 +22,11 @@
 import sys
 from functools import partial
 from PyQt4 import QtGui
-from PyQt4.QtGui import QPalette, QColor
+from PyQt4.QtGui import QPalette
 from picard import config
 from picard.const import PICARD_URLS
 from picard.file import File
-from picard.script import ScriptParser, SyntaxError, ScriptError, UnknownFunction
+from picard.script import ScriptParser, SyntaxError, ScriptError
```

Comment:
@mineo: this cleanup is totally unrelated to the rest of the PR imho. Can you make a standalone import cleanup PR ? I agree with those trivial changes of course, but it is better to not mix them with feature PRs.